openssl-errors/tests/test.rs

```diff
@@ -16,6 +16,7 @@ openssl_errors::openssl_errors! {
 }
 
 #[test]
+#[allow(clippy::single_char_pattern)]
```

Comment:
Probably better to just fix the thing it's complaining about.

Reply:
When I remove the clippy-`allow`'s, I get errors in the CI, but not locally (running the same command `cargo clippy --all --all-targets`). I'll try to fix the errors blind, then.